Tests/NIOHTTPClientTests/SwiftNIOHTTPTests.swift

```diff
@@ -169,6 +169,17 @@ class SwiftHTTPTests: XCTestCase {
 
         response = try httpClient.get(url: "http://localhost:\(httpBin.port)/redirect/https?port=\(httpsBin.port)").wait()
         XCTAssertEqual(response.status, .ok)
+        
+        response = try httpClient.get(url: "https://httpbin.org/redirect-to?url=https%3A%2F%2Fwww.httpbin.org%2Fheaders").wait()
```

Comment:
IMO this should be a new, separate test case. Especially since we're using two separate servers.